src/host/ut_host/OutputCellIteratorTests.cpp

```diff
@@ -102,8 +102,7 @@ class OutputCellIteratorTests
     {
         SetVerifyOutput settings(VerifyOutputSettings::LogOnlyFailures);
 
-        TextAttribute attr;
-        attr.SetFromLegacy(FOREGROUND_RED | BACKGROUND_BLUE);
+        const TextAttribute attr(FOREGROUND_RED | BACKGROUND_BLUE);

```

Comment:
(nit: "most vexing parse" comment again)

Reply:
As I mentioned above, this is just the style of the surrounding code. Notice the settings variable above, and the iterator and view a couple lines below.